Description: Fix pkg-config variable typo.
 The variable set by PKG_PROG_PKG_CONFIG which points to the pkg-config
 utility is PKG_CONFIG, not PKGCONFIG.  The latter was previously used
 when trying to detect the presence of the MagickWand library when Window
 Maker was built using --enable-magick.
Origin: http://repo.or.cz/wmaker-crm.git/commitdiff/d13b78b
Last-Update: 2018-08-08

--- a/m4/wm_imgfmt_check.m4
+++ b/m4/wm_imgfmt_check.m4
@@ -287,14 +287,14 @@
         [wm_cv_libchk_magick=no
          dnl First try to get the configuration from either pkg-config (the official way)
          dnl or with the fallback MagickWand-config
-         AS_IF([test "x$PKGCONFIG" = "x"],
+         AS_IF([test "x$PKG_CONFIG" = "x"],
              [AC_PATH_PROGS_FEATURE_CHECK([magickwand], [MagickWand-config],
                  [wm_cv_libchk_magick_cflags=`$ac_path_magickwand --cflags`
                   wm_cv_libchk_magick_libs=`$ac_path_magickwand --ldflags`
                   wm_cv_libchk_magick=magickwand])],
-             [AS_IF([$PKGCONFIG --exists MagickWand],
-                 [wm_cv_libchk_magick_cflags=`$PKGCONFIG --cflags MagickWand`
-                  wm_cv_libchk_magick_libs=`$PKGCONFIG --libs MagickWand`
+             [AS_IF([$PKG_CONFIG --exists MagickWand],
+                 [wm_cv_libchk_magick_cflags=`$PKG_CONFIG --cflags MagickWand`
+                  wm_cv_libchk_magick_libs=`$PKG_CONFIG --libs MagickWand`
                   wm_cv_libchk_magick=pkgconfig])])
          AS_IF([test "x$wm_cv_libchk_magick" = "xno"],
              [AS_IF([test "x$enable_magick" != "xauto"],
